packages/astro/src/toolbar/index.ts

```diff
@@ -0,0 +1,5 @@
+import type { DevToolbarApp } from '../@types/astro.js';
+
+export function defineToolbarApp(app: DevToolbarApp) {
+	return app;
+}
```

Comment:
Since it's intended for external usage, and possibly build systems. I figured `astro/toolbar` would be better, otherwise you can't build using TypeScript, you need to use `external` etc. I'm expecting that we'll do the same with `astro/integration` if we have exports there.